proxy/Cargo.toml

```diff
@@ -21,6 +21,7 @@ nonempty = "0.2"
 pico-args = "0.3"
 pretty_env_logger = "0.3"
 rand = "0.7"
+secstr = "0.3"
```

Comment:
:shushing_face: 

@kim This actually brings up a question for me: should re-export `SecUtf8` since we're implementing `Pinentry` on it. That means the versioning here has to match. That sound reasonable to you?

Reply:
Yeah right... crate hell

Reply:
:fire: :smiling_imp: :fire:

Reply:
https://github.com/radicle-dev/radicle-link/pull/184